docker-compose.yaml

```
@@ -0,0 +1,70 @@
+version: "3.8"
+services:
+  ceph-spdk:
+    image: ceph/spdk-ubi9
+    profiles:
+      - build
+    build:
+      context: spdk/
+      dockerfile: ../Dockerfile.spdk
+      target: ceph-spdk
+      args:
+        SPDK_VERSION: v23.01
+        PKGDEP_ARGS: --rbd
+        CONFIGURE_ARGS: >-
+          --with-rbd
+          --disable-tests
+          --disable-unit-tests
+          --disable-examples
+  ceph-cluster:
+    image: ceph-cluster
+    container_name: ceph-cluster
+    build:
+      context: .
+      dockerfile: Dockerfile.ceph
+    environment:
```

```suggestion
    depends_on:
      change-vol-ownership:
        condition: service_completed_successfully
    environment:
```

Second part of fix ceph conf directory ownership